src/components/Panel3/Panel3.jsx

```diff
@@ -76,7 +77,7 @@ function Panel3({ blocks }) {
 
       setTableTxRowElements((prev) => [newRows, ...prev].slice(0, MAX_ROWS));
     } catch (err) {
-      console.error(err);
+      Log(err);
```

Comment:
It should be Error(err) rather Log(err) I think

Reply:
ok

Reply:
`Error` is a keyword 
![image](https://user-images.githubusercontent.com/37145078/156313457-cac1e820-cf59-4724-bd97-ddd9acbf706a.png)